drydock/plugin.py

```diff
@@ -75,11 +75,10 @@ def get_init_tasks():
 CORE_SYNC_WAVES_ORDER: SYNC_WAVES_ORDER_ATTRS_TYPE = {
     "drydock-upgrade-lms-job": 50,
     "drydock-upgrade-cms-job": 51,
-    "lms-lifecycle-enabled": 100,
-    "cms-lifecycle-enabled": 100,
     "lms-debug": 50,
     "cms-debug": 50,
     "ingress-debug": 200,
+    "deployments:openedx-apps": 100,
```

Comment:
```suggestion
    "deployments:post-init-apps": 100,
```
Does it sound better for you? Replace where it applies